<?xml version="1.0"?>

<launch>
	<node name="GoalSender" pkg="human_aware_navigation" type="GoalSender" output="screen"/>
</launch>
